extension-sdk/src/service-worker/index.js

```
@@ -0,0 +1,982 @@
+              break;
+            case 'saveUISettings':
+              await setToSyncStorage('user_logged_obj', message.data);
+              sendResponse({ success: true });
+              break;
+            case 'getUserIntegration':
+              await getUserIntegration();
+              sendResponse({ success: true });
+              break;
+            case 'getSettings':
+              const uiSettings = await getUiSettings();
+              sendResponse({ uiSettings });
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)